includes/framework/QDatabaseBase.class.php

```diff
@@ -1299,5 +1299,7 @@ public function __get($strName) {
 		const Float = "Float";
 		/** Boolean */
 		const Bit = "Bit";
+		/** JSON (for Postgresql currently) */
+		const Json = "JSON";
```

Comment:
I believe, JSON is coming into more and more databases anyway. However, I think that it could do better in another class. 
Let me see about that.